src/test/java/com/seals/camplanner/location/controllers/LocationControllerTest.java

```diff
@@ -43,17 +46,19 @@
     @Autowired
     private LocationService locationService;
 
+    private ModelMapper modelMapper = new ModelMapper();
```

Comment:
Since the `ModelMapper` is declared as a Spring bean it can be `@Autowired` here.